gradle/libs.versions.toml

```diff
@@ -13,17 +15,24 @@ hiltExt = "1.2.0"
 ksp = "2.0.0-1.0.21"
 dagger = "2.51.1"
 androidxHiltNavigationCompose = "1.2.0"
+mockitoCore = "3.11.2"
 okhttp = "4.12.0"
 retrofit = "2.9.0"
 retrofitKotlinxSerializationJson = "1.0.0"
 kotlinxSerializationJson = "1.6.3"
 coil = "2.6.0"
+robolectric = "4.9.1"
 room = "2.6.1"
 paging = "3.3.0"
 androidxNavigation = "2.8.0-beta05"
+runner = "1.6.1"
+junitKtx = "1.2.1"
 
 [libraries]
 androidx-core-ktx = { group = "androidx.core", name = "core-ktx", version.ref = "coreKtx" }
+androidx-runner = { module = "androidx.test:runner", version.ref = "runner" }
```

Comment:
We already have had the hilt dependencies: `hiltTesting = { module = "com.google.dagger:hilt-android-testing", version.ref = "dagger" }`.